src/core/Expander/Expander.baseStyles.tsx

```diff
@@ -33,16 +33,20 @@ export const baseStyles = withSuomifiTheme(
     position: relative;
     display: block;
     width: 100%;
+    font-size: ${theme.typography.bodySemiBold};
+    min-height: 60px;
```

Comment:
Would it be possible to use `spacing.xxxl` from the tokens? And little later on for the `height` and `width` of the `.fi-expander_title-icon`

Or where is the fine line when not to use tokens? At least from the naming it might be confusing to use spacing, when we are talking about component height. 
But the ideal would be to use existing tokens is possible to keep different sizes consistent 🤔

Reply:
That's what I was thinking with these values, and it is a good question indeed.

Reply:
I would not use spacing tokens for size. Size should consist of content + borders + spacing. Which all can be defined by tokens if we play by the book. E.g. Typography token with line height set + border token or default style with constant width + token for spacing => always a deterministic set of possible heights.

Reply:
Let's use the hardcoded pixel value for now and change to use tokens at later alignments.